nodeJS/authentication/Authentication.md

```diff
@@ -289,7 +289,7 @@ If you insert this code somewhere between where you instantiate the passport mid
 
 Now, let's go back and learn how to securely store user passwords so that if anything ever goes wrong, or if someone gains access to our database, our user passwords will be safe.  This is _insanely_ important, even for the simplest apps, but luckily it's also really simple to set up.
 
-First `npm install bcryptjs`. There is another module called `bcrypt` that does the same thing, but it is written in C++ and is sometimes a pain to get installed. The C++ `bcrypt` is technically faster, so in the future it might be worth getting it running, but for now the modules work the same so we can just use `bcryptjs`.
+First `npm install bcryptjs`. There is another module called `bcrypt` that does the same thing, but it is written in C++ and is sometimes a pain to get installed. The C++ `bcrypt` is technically faster, so in the future, it might be worth getting it running, but for now, the modules work the same so we can just use `bcryptjs`.
```

Comment:
I'm not sure this extra comma is necessary

Reply:
but for now or so in the future?

Reply:
Grammarly told me so but its not 100% accurate

Reply:
let me know...

Reply:
Sorry, "so in the future". It just feels like that's unnecessarily breaking up that piece of the sentence.

Reply:
Ok. changed that.